src/runtime_src/driver/xclng/xrt/user_aws/shim.h

```diff
@@ -189,7 +189,7 @@ class RangeTable {
 
 
         // Bitstreams
-        int xclGetXclBinIdFromSysfs(uint64_t &xclbinid);
+        int xclGetXclBinUuidFromSysfs(uuid_t &xclbinid);
```

Comment:
const function?

Reply:
Change to "int xclGetXclBinUuidFromSysfs(uuid_t &xclbinid) const;" ?